sql/core/src/main/scala/org/apache/spark/sql/internal/BaseSessionStateBuilder.scala

```diff
@@ -166,13 +169,14 @@ abstract class BaseSessionStateBuilder(
    *
    * Note: this depends on the `conf` and `catalog` fields.
    */
-  protected def analyzer: Analyzer = new Analyzer(catalog, v2SessionCatalog, conf) {
-    override val extendedResolutionRules: Seq[Rule[LogicalPlan]] =
-      new FindDataSourceTable(session) +:
-        new ResolveSQLOnFile(session) +:
-        new FallBackFileSourceV2(session) +:
-        DataSourceResolution(conf, this.catalogManager) +:
-        customResolutionRules
+  protected def analyzer: Analyzer =
+    new Analyzer(catalogManager, conf) {
```

Comment:
super nit: we can keep the indentation unchanged? i.e. still put the `new Analyzer` in the above line.

Reply:
Done.